### ISSUE TITLE
Transfer Learning - train on overseas data

### ISSUE THREAD

**Issue by LennoxC** [LennoxC/convcnp-assim-nz#11]
Use transfer learning, and initially train the data assimilation task on data from across the entire globe, and then fine-tune the model to NZ later? Would help with underfitting issues.

Global data doesn't have to use the exact same format - as long as the task has resemblance to the NZ context, starting with these weights would be better than random initialisation.

From Tristan:
- The main idea is to train on ERA5, then fine tune with NZRA. 
- So ERA5 + Himawari satellite data + observations, then fine tune with NZRA (?)

Other ideas:
- batch based training (grouped by num targets) initially, then fine-tune with gradient accumulation later?
